prettier.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  semi: true,
```

Comment:
You can omit this, it is `true` by default

Reply:
Indeed. But let's keep it as it's explicit and can serve as a guide for anyone who want to check. I believe not everyone knows what're the default values of prettier's options.